lessons/misc-quick-reference/cs-string.md

```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
 var mathProblem = $"What do you get when you multiply six by 9: {6 * 9}"
 ```
 
----
-
 ### String length
 
----
+We can get the length of a string by using the length property.
+
+```C#
+var name = "Suncoast Developers Guild";
+
+var nameLength = name.Length;
+
+Console.WriteLine(nameLength);
+//Output: 25;
```

Comment:
I'd prefer that all comments have at least a single space.

```
// Output: 25
```

And there wouldn't be a `;` on that output...

Reply:
I corrected this example and added single spaces to all comments throughout my additions.